### ISSUE TITLE
Polish translation

### ISSUE THREAD

**Issue by pdurbin** [GlobalDataverseCommunityConsortium/dataverse-language-packs#26]
There's a Polish translation at https://github.com/CeON/dataverse/pull/29 that we should add.

Thanks to @madryk and @MrK191 !

**Comment by 4tikhonov** [comment]
Was it tested for the latest version of Dataverse?

**Comment by pdurbin** [comment]
@4tikhonov I believe it's for Dataverse 4.10.1. But that's fine. It would be a good starting point for anyone who would like to help maintain it going forward. 😄

**Comment by 4tikhonov** [comment]
Ok, I have something nice to show next week about keeping all translations up-to-date.

**Comment by 4tikhonov** [comment]
<img width="1423" alt="Screen Shot 2019-06-13 at 2 17 11 PM" src="https://user-images.githubusercontent.com/704075/59431829-212b6f80-8de6-11e9-9d0d-2cc22ab6322c.png">

**Comment by amberleahey** [comment]
thanks @4tikhonov this is great to see, looks like the Weblate tool and it's integrated with Github! We did notice some blank strings in English bundle files causes some issues (flagging empty strings for translation), but something to improve upon for sure.

**Comment by 4tikhonov** [comment]
@amberleahey, yes, in the default configuration it's not very convenient so we've made a few modifications and running it next to Dataverse instance.

**Comment by pdurbin** [comment]
@4tikhonov this looks great! I can't wait to see your talk!

@4tikhonov and @amberleahey should we set up some automation around this at https://jenkins.dataverse.org ? Our config is at https://github.com/IQSS/dataverse-jenkins and I would be happy to help configure as many jobs as you need.
